### PR TITLE
Force clean up command

### PR DESCRIPTION
**Pull request by D4nte**
For when everything goes wrong

### CODE REVIEW

File: tests/force_clean_env.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# Environment is (at least half) started
+
+# SIGKILL create-comit-env, the env is not cleaned up
+kill -9 $PID;
+wait $PID || true; # Working around the `set -e` as SIGKILL makes it return bad code
+
+# Check the environment is still up after SIGKILL'ng create-comit-app
+if [ "$(check_containers)" -ne 0 ]; then
+      echo "FAIL: The environement is not up for some reason."
```
Comment (luckysori) [inline tests/force_clean_env.sh]: :wrench: 
```suggestion
      echo "FAIL: The environment is not up for some reason."
```

File: tests/force_clean_env.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  echo "FAIL: Containers were not stopped.";
+  exit 1;
+fi
+
+if [ -d "$HOME/.create-comit-app" ]; then
+  echo "FAIL: ~/.create-comit-app was not deleted.";
+  exit 1;
+fi
+
+if test $(docker network ls |grep -n create-comit-app); then
```
Comment (luckysori) [inline tests/force_clean_env.sh]: :wrench:
```suggestion
if test $(docker network ls | grep -n create-comit-app); then
```

File: tests/force_clean_env.sh [hunk 3]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+set -e
+
+CWD=${0%/tests/*.sh}
```
Comment (tcharding) [inline tests/force_clean_env.sh]: Add comment please, this is cool but obscure.

File: tests/force_clean_env.sh [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      ENV_UP=true;
+      TIMEOUT=0
+    else
+      sleep 1;
+      TIMEOUT=$((TIMEOUT-1));
+    fi
+done
+
+if ! $ENV_UP; then
+  echo "FAIL: Environment never started";
```
Comment (tcharding) [inline tests/force_clean_env.sh]:
```suggestion
  echo "FAIL: Environment failed to start";
```

**Pull request merged**